src/super_gradients/common/env_variables.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+# Infra
+DECI_PLATFORM_TOKEN = os.getenv("DECI_PLATFORM_TOKEN")
```

Comment:
Lets switch this to a class as you suggested (in order to avoid problems when a user would set the env vars after import of any of these)

Reply:
Changed: 
https://github.com/Deci-AI/super-gradients/blob/121b61114d08a0e33a1f92f0bde684cf72951ec8/src/super_gradients/common/environment/env_variables.py#L1-L39